### PR TITLE
[BISERVER-14069] - Fixed Language Translation Properties Files Not Pi…

### PR DESCRIPTION
**Pull request by RPAraujo**
…cked up from PRPTs when running on Server

@pentaho/tatooine @tmorgner please review

### CODE REVIEW

File: core/src/main/java/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java [hunk 1]
```diff
@@ -390,6 +390,7 @@ private Serializable computeCacheKey( final MasterReport report ) throws BeanExc
       sourceKey = computeDefSourceKey( report, definitionSource );
     }
     final HashMap<String, String> params = new HashMap<>();
+    params.put("report-locale", report.getReportEnvironment().getLocale().toString() );
```
Comment (pamval) [inline core/src/main/java/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java]: Check style issue, please fix

Comment (tmorgner) [inline core/src/main/java/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java]: Adding the locale information into the same collection as the report's parameters will clash the moment a user has a report parameter named "report-locale". Add that information as new element into the "key" array list below. Remember: The only requirement of the result of this function is that it is serializable and that it uniquely identifies a report with all its configuration details and parameters. So adding the locale string down there is safe and a lot better than risking namespace clashes.

Comment (RPAraujo) [reply core/src/main/java/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java]: Done

Comment (RPAraujo) [reply core/src/main/java/org/pentaho/reporting/platform/plugin/output/CachingPageableHTMLOutput.java]: Done

**Pull request merged**